.github/workflows/pull_request.yml

```diff
@@ -109,10 +109,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Set up Go 1.19
+      - name: Set up Go 1.20
```

Comment:
Can you please update the version on line no 84 as well?

Reply:
apologies will get that done